misc-utils/lsfd-sock-xinfo.c

```diff
@@ -360,7 +359,7 @@ static void send_diag_request(int diagsd, void *req, size_t req_size,
 		.msg_iov = iovecs,
 	};
 
-	alignas(void *) uint8_t buf[8192];
+	__attribute__((aligned(sizeof(void *)))) uint8_t buf[8192];
```

Comment:
See https://github.com/util-linux/util-linux/pull/2511#discussion_r1339605820 :-)

Of course, I have no objection. 

It would be nice if the standards and extensions a contributor assumes were written explicitly in a document (Documentation/howto-contribute.txt?) when contributing to util-linux.